src/components/daydream-controls.js

```diff
@@ -89,40 +90,33 @@ module.exports.Component = registerComponent('daydream-controls', {
     if (isPresent) { this.injectTrackedControls(); } // inject track-controls
   },
 
-  onGamepadConnected: function (evt) {
-    // for now, don't disable controller update listening, due to
-    // apparent issue with FF Nightly only sending one event and seeing one controller;
-    // this.everGotGamepadEvent = true;
-    // this.removeControllersUpdateListener();
-    this.checkIfControllerPresent();
-  },
-
-  onGamepadDisconnected: function (evt) {
-    // for now, don't disable controller update listening, due to
-    // apparent issue with FF Nightly only sending one event and seeing one controller;
-    // this.everGotGamepadEvent = true;
-    // this.removeControllersUpdateListener();
+  onGamepadConnectionEvent: function (evt) {
+    this.everGotGamepadEvent = true;
```

Comment:
no because we expect gamepadconnected next time since we're getting events, right?  so we don't need to poll every tick